src/server_manager/web_app/app.ts

```diff
@@ -550,9 +540,9 @@ export class App {
   }
 
   private async handleConnectDigitalOceanAccountRequest(): Promise<void> {
-    let accessToken: string;
+    let digitalOceanAccount: DigitalOceanAccount;
     try {
-      accessToken = await this.runDigitalOceanOauthFlow();
+      digitalOceanAccount = await this.runDigitalOceanOauthFlow();
```

Comment:
I would prefer is the oauth code returned the credentials. Then you call connectDigitalOceanAccount here.

The oauth is a low level concept, independent of our app.